peps/conf.py

```diff
@@ -24,6 +24,7 @@
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
     "sphinx.ext.githubpages",
+    "sphinxcontrib.mermaid",
```

Comment:
This is not working as expected. The mermaid diagram is not rendered. Might be hitting https://github.com/mgaitan/sphinxcontrib-mermaid/issues/124, but I tried different plugin orderings and it didn't change much.